build.sh

```diff
@@ -1,123 +1,16 @@
 #!/usr/bin/env bash
 
-# adding dotnet to the path. it is needed to run toolset csc.
-parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
-dotnet_path=$parent_path/dotnetcli
-export PATH=$PATH:$dotnet_path
+source="${BASH_SOURCE[0]}"
 
-usage()
-{
-  echo "build.sh [ Debug(default) | Release ]"
-}
+# resolve $SOURCE until the file is no longer a symlink
+while [[ -h $source ]]; do
+  scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+  source="$(readlink "$source")"
 
-if [ "$1" == "-?" ] || [ "$1" == "-h" ]; then
-  usage
-  exit
-fi
+  # if $source was a relative symlink, we need to resolve it relative to the path where the
+  # symlink file was located
+  [[ $source != /* ]] && source="$scriptroot/$source"
+done
 
-if [ $# -eq 0 ]
-  then
-    Configuration="Debug"
-  else
-    Configuration=$1
-fi
-
-BuildVersion="preview2-$(date '+%y%m%d')-1"
-Version="<default>"
-Restore="true"
-echo "Configuration=$Configuration."
-echo "Restore=$Restore."
-echo "Version=$Version."
-echo "BuildVersion=$BuildVersion."
-
-dotnetExePath="dotnetcli/dotnet"
-
-if [ "$Version" = "<default>" ]; then
-  Version=$(head -n 1 "DotnetCLIVersion.txt")
```

Comment:
How do we now control/update the CLI version we need to install/use in corefxlab?

Also, is the plan to remove DotnetCLIVersion.txt?
https://github.com/dotnet/corefxlab/blob/154fafa095dd6159867c4aeb096d31c60ae96ad0/DotnetCLIVersion.txt#L1

Reply:
Same as in dotnet/runtime and the rest of the arcade repos, this will be controlled via the root's global.json file.

Reply:
> Also, is the plan to remove DotnetCLIVersion.txt?

Yes that file should be removed.